tests/packages/subproject/pyproject.toml

```diff
@@ -5,3 +5,8 @@
 [build-system]
 build-backend = 'mesonpy'
 requires = ['meson-python']
+
+[tool.meson-python.wheel]
+# Meson versions before 1.3.0 install data files in {datadir}/{project name}/,
+# later versions install in the more correct {datadir}/{dubproject name}/
+exclude = ['{datadir}/*/data.txt']
```

Comment:
This moves install location markers into the public API, which is something that may constrain us in the future, and would need to be documented. I think it'd be better to match on paths inside the wheel, so for this it'd be `subproject-*.data/dep/data.txt'`. That pattern should not be variable, but is given by the [wheel spec](https://packaging.python.org/en/latest/specifications/binary-distribution-format/#file-contents)

Reply:
Another issue with using wheel locations is that it makes globbing to exclude whole subdirectories much less useful.

Reply:
Ah yes, of course. I think the `{ }` introspection data is stable, so it'll probably be fine - I wanted to raise it to be certain. 

Let me resolve this thread.